drivers/driver_functions_test.go

```diff
@@ -0,0 +1,50 @@
+package drivers
+
+import (
+	"fmt"
+	"github.com/rancher/go-rancher/v2"
+	"os"
+	"testing"
+	"time"
+)
+
+func TestScaleService(t *testing.T) {
```

Comment:
Ok, forget this test for now. I think we are going to have to do end-to-end testing of this as validation tests.
Instead, right tests that exercise the framework end-to-end without having to call cattle.

To do this, you should write a "test" or "mock" driver and register it as a driver in your Test setup. Then, start up your service in your test and make HTTP requests to it to generate and execute webhooks.